healthcheck/core/src/main/java/org/apache/felix/hc/core/impl/executor/async/cron/CronSchedulerTask.java

```diff
@@ -0,0 +1,53 @@
+package org.apache.felix.hc.core.impl.executor.async.cron;
+
+import org.slf4j.Logger;
```

Comment:
Same here: SLF4J Logger vs. OSGi Logger?

Reply:
Since this class is not leveraging SCR, I used slf4j